packages/amplify_datastore/ios/Classes/DataStoreHubEventStreamHandler.swift

```diff
@@ -0,0 +1,133 @@
+                            self.sendError(description: castingError, eventName: "subscriptionsEstablished")
+                        }
+                    case HubPayload.EventName.DataStore.syncQueriesStarted :
+                        do {
+                            let syncQueriesStarted =  try FlutterSyncQueriesStartedEvent(payload: payload)
+                            self.sendEvent(flutterEvent: syncQueriesStarted.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: "syncQueriesStarted")
+                        }
+                    case HubPayload.EventName.DataStore.modelSynced :
+                        do {
+                            let modelSynced =  try FlutterModelSyncedEvent(payload: payload)
+                            self.sendEvent(flutterEvent: modelSynced.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: "modelSynced")
+                        }
+                    case HubPayload.EventName.DataStore.syncQueriesReady :
+                        do {
+                            let syncQueriesReady =  try FlutterSyncQueriesReadyEvent(payload: payload)
+                            self.sendEvent(flutterEvent: syncQueriesReady.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: "syncQueriesReady")
+                        }
+                    case HubPayload.EventName.DataStore.ready :
+                        do {
+                            let ready =  try FlutterReadyEvent(payload: payload)
+                            self.sendEvent(flutterEvent: ready.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: "ready")
+                        }
+                    case HubPayload.EventName.DataStore.outboxMutationEnqueued :
+                        do {
+                            guard let outboxMutationEnqueued = payload.data as? OutboxMutationEvent else {
+                                      throw FlutterDataStoreError.hubEventCast
+                                  }
+                            let modelName = outboxMutationEnqueued.modelName
+                            let schema = try FlutterDataStoreRequestUtils.getModelSchema(modelSchemas: self.flutterModelRegistration?.modelSchemas ?? [:], modelName: modelName)
+
+                            let flutterOutboxMutationEnqueued = try FlutterOutboxMutationEnqueuedEvent(outboxMutationEnqueued: outboxMutationEnqueued, eventName: payload.eventName, schema: schema)
+                            self.sendEvent(flutterEvent: flutterOutboxMutationEnqueued.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: payload.eventName)
+                        }
```

Comment:
for `eventName` in the sendError, can we use one of `payload.eventName` or manually provided name, but not mix both?

Reply:
Replaced this with...
```
 self.sendError(hubEvent: shortEventName(eventName: payload.eventName))
```
...to trim off 'DataStore.' prefix.